From 47e5b1e80979631b6476e99f78d7a6ae17d1a8ff Mon Sep 17 00:00:00 2001
From: Tom Stellard <tstellar@redhat.com>
Date: Tue, 21 Feb 2017 23:58:19 +0000
Subject: [PATCH] test: Hard-code have_zlib to "1" in lit.site.cfg.in

The stand-alone clang builds don't do zlib detection at all, so
have_zlib is always false, which is incorrect, since we build llvm
with zlib enabled.

This fixes Driver/nozlibcompress.c
---
 test/lit.site.cfg.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/lit.site.cfg.in b/test/lit.site.cfg.in
index f368c99..5ca4d03 100644
--- a/test/lit.site.cfg.in
+++ b/test/lit.site.cfg.in
@@ -14,7 +14,7 @@ config.clang_tools_dir = "@CLANG_TOOLS_DIR@"
 config.host_triple = "@LLVM_HOST_TRIPLE@"
 config.target_triple = "@TARGET_TRIPLE@"
 config.llvm_use_sanitizer = "@LLVM_USE_SANITIZER@"
-config.have_zlib = "@HAVE_LIBZ@"
+config.have_zlib = "1"
 config.clang_arcmt = @ENABLE_CLANG_ARCMT@
 config.clang_staticanalyzer = @ENABLE_CLANG_STATIC_ANALYZER@
 config.clang_examples = @ENABLE_CLANG_EXAMPLES@
-- 
2.9.3
